### ISSUE TITLE
[Synthese] Carte vide si paramètre AREA_AGGREGATION_ENABLED = true

### ISSUE THREAD

**Issue by jpm-cbna** [PnX-SI/GeoNature#3180]
**Version**
v2.14.2

**Description du bug**
Lorsque que le paramètre `AREA_AGGREGATION_ENABLED`, section `[SYNTHESE]` du fichier de config `geonature_config.toml` vaut `true`, les recherches dans la Synthese n'affiche rien sur la carte. Un clic sur l'icône 🛈 de la liste d'observation provoque l'affichage du message: 
   
   > L'observation selectionnée n'est présente dans aucune maille - passez en mode 'point' pour la 
   localiser

Rien ne s'affiche sur la carte.

**Comportement attendu**
Les observations devrait s'afficher sur la carte sous forme de maille colorée en fonction du nombre d'observations qu'elles contiennent.

**Comment reproduire**
Modifier le fichier `geonature_config.toml` avec ce paramètrage:
```properties
[SYNTHESE]
    AREA_AGGREGATION_ENABLED = true
    AREA_AGGREGATION_TYPE = "M5"
```

**Comment by camillemonchicourt** [comment]
Oui ça me parle que l'agrégation des observations dans la synthèse en mode Maille ne fonctionne plus dans la Synthèse. 🤔

**Comment by ch-cbna** [comment]
Après reproduction du problème, nous nous sommes aperçus que le message d'erreur n'apparaissait plus et que le fonctionnement des mailles fonctionnait bien. Pour l'instant nous gardons ce ticket ouvert au cas où quelqu'un tombe sur cette erreur.

**Comment by camillemonchicourt** [comment]
Oui, en fait, je me suis embrouillé, pas de soucis identifié non plus sur le serveur de DEMO.
